l10n_es_aeat_sii_oca/models/account_move.py

```diff
@@ -1106,9 +1106,16 @@ def _sii_invoice_dict_not_modified(self):
         content_sent = json.loads(self.sii_content_sent)
         return to_send == content_sent
 
-    def action_post(self):
-        res = super(AccountMove, self).action_post()
-        for invoice in self.filtered("sii_enabled"):
+    def post(self):
+        res = super().post()
+        for invoice in self:
```

Comment:
No se si sería mejor realizar un filter en el for de esta manera en vez del if:

`for invoice in self.filtered(lambda x: x.sii_enabled and x.is_invoice())`

utilizando la función de 'is_invoice' podemos aligerar el código de comprobación de tipos (es como lo está realizando odoo actualmente según he ido viendo).

Reply:
Pues sí, no recordaba esa función, gracias. ¡Cambiado!